ares/fc/cartridge/board/bandai-74161.cpp

```diff
@@ -0,0 +1,64 @@
+struct Bandai74161 : Interface {
+  static auto create(string id) -> Interface* {
+    if(id == "BANDAI-74161" ) return new Bandai74161(Revision::B74161);
```

Comment:
Maybe, but note that there are more board names in ares right now with names different than those on NesCartDb. If those names are preferable I can change that.

Reply:
As this name system is often used by other emulators (MAME, Mesen, Nestopia being some of the examples), I personally think it would be good if this system could be followed as much as reasonably possible.

Reply:
I wouldn't get too hung up on naming for now. I've been looking at nescartdb for years and it's not an easy thing to come up with a consistent solution for. Company+chips does seem the most straightforward... but there are a few boards with like 10 discrete logic chips on them and that would be a pretty long name. As long as boards are being appropriately split due to differences, we can always come back to names.